vitrivr-engine-index/src/main/kotlin/org/vitrivr/engine/database/jsonl/retrievable/JsonlRetrievableInitializer.kt

```diff
@@ -2,22 +2,43 @@ package org.vitrivr.engine.database.jsonl.retrievable
 
 import org.vitrivr.engine.core.database.retrievable.RetrievableInitializer
 import org.vitrivr.engine.database.jsonl.JsonlConnection
+import org.vitrivr.engine.database.jsonl.LOGGER
+import java.io.File
+import java.io.IOException
 
 class JsonlRetrievableInitializer(private val connection: JsonlConnection) : RetrievableInitializer {
 
+    private val retrievableFile = File(connection.schemaRoot, "retrievables.jsonl")
+    private val connectionFile = File(connection.schemaRoot, "retrievable_connections.jsonl")
+
     override fun initialize() {
-        TODO("Not yet implemented")
+        try {
+            connection.schemaRoot.mkdirs()
+            retrievableFile.createNewFile()
+            connectionFile.createNewFile()
+        } catch (ioe: IOException) {
+            LOGGER.error(ioe) { "Cannot initialize '${connection.schemaRoot.absolutePath}'" }
+        } catch (se: SecurityException) {
+            LOGGER.error(se) { "Cannot initialize '${connection.schemaRoot.absolutePath}'" }
+        }
     }
 
     override fun deinitialize() {
-        TODO("Not yet implemented")
+        /* nop */
```

Comment:
Why is this a No-OP? Shouldn't this simply delete the files?

Reply:
I see, sure.